### ISSUE TITLE
Fix pyflakes for runner

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6992]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#6992|
|Type|enhancement|
|Created|2014-02-27 12:19:47Z|
|Branch|https://github.com/twisted/twisted/tree/pyflakes-procmon-6992|

Attachments:

* [6992-1.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/938/938c7cba510ac29604b073fb682d9d5d8c62d489/fa1e421f842dffacd2710c40c607657370158a37.diff) (1663 bytes) - added by adiroiban on 2014-02-27 12:30:52Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6992 6992
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__ 
branch__branches_pyflakes_procmon_6992 branches/pyflakes-procmon-6992
branch_author__rwall rwall
status__closed closed
resolution__fixed fixed
component__runner runner
keywords__None None
time__1393503587000000 1393503587000000
changetime__1474198049187875 1474198049187875
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban commented|
|-|-|

This code looks strange.

There are no tests for it and I have no idea if RPCServer is working in master.

It uses a undefined rpcVersions and 'p' variables.

I set the init rpcVersions to self.rpcVersions so that it will be availabe in startService.
That 'p' variables tells me nothing so I have removed it.

imported warning was not used.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)|@wallrj set owner to @wallrj<br>@wallrj set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_3"></a>|@wallrj set owner to @adiroiban<br>@wallrj set status to `new`|
|-|-|

Thanks adiroban,

The missing rpcVersions variable seems to stem from some refactoring which took place between r4360 and r8812.
Your changes to inetdtap look correct, but I don't think it would be right to merge those changes without accompanying tests.

See [#6873](https://github.com/twisted/twisted/issues/6873) for more comments about fixing or deprecating inetdtap. Please add your thoughts to that ticket and perhaps you could help resolve it.

The change to procmon.py looks fine.

Would you like me to merge just the procmon change for now?

Thanks again.

-RichardW.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban commented|
|-|-|

Since the code is already bad (for a long time) and we know it, here I just tried to clean it up for pyflakes so that we don't have pyflakes errors until [[#6873](https://github.com/twisted/twisted/issues/6873)](https://github.com/twisted/twisted/issues/6873) will be fixed.

I will try to fix the other part in #6873... but I think that is safe to merge current  changes so that we can have at least a clean pyflakes report.

Please don't merge the patch as I want to update to for using the namedAny method and avoid using the stupid comment.

Thanks

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban removed owner|
|-|-|

OK. I have checked to code. Please apply only procmon changes as the other changes should be fixed in [#6873](https://github.com/twisted/twisted/issues/6873)

Thanks

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_6"></a>|@wallrj commented|
|-|-|

(In [42132]) Branching to 'pyflakes-procmon-6992'

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_7"></a>|@wallrj commented|
|-|-|

(In [42133]) Apply just the procmon changes from 6992-1.diff by adiroban. Refs [#6992](https://github.com/twisted/twisted/issues/6992)

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_8"></a>|@wallrj set status to `closed`|
|-|-|

(In [42135]) Merge pyflakes-procmon-6992

Author: adiroiban
Reviewer: rwall
Fixes: [#6992](https://github.com/twisted/twisted/issues/6992)

Remove unused imports from twisted.runner.procmon. Remaining twisted.runner
pyflakes warnings will be dealt with in [#6873](https://github.com/twisted/twisted/issues/6873).

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_9"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
